ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh

```diff
@@ -6,6 +6,7 @@ set -o pipefail
 
 export AWS_SHARED_CREDENTIALS_FILE=${CLUSTER_PROFILE_DIR}/.awscred
 export AZURE_AUTH_LOCATION=${CLUSTER_PROFILE_DIR}/osServicePrincipal.json
+export IBMCLOUD_SHARED_CREDENTIALS_FILE=${CLUSTER_PROFILE_DIR}/ibmcloud.json
```

Comment:
Note mostly for me but FYI as well. This should exist in the vault. So the file name `ibmcloud.json` should be mapped in the bootstrap config as well.

Reply:
I think maybe instead we would just use an API Key environment variable, not a json file, since I don't believe we have any other data to inject into it.

I'll look a little, but think this will simply end up being a variable for an API Key, pending what is put in vault.

Reply:
This will likely end up something like this, given the requirement of the IBM Cloud Terraform provider
```
export IC_API_KEY="$(< ${CLUSTER_PROFILE_DIR}/openshift-e2e-ci-apikey)"
```

Reply:
Let's go for the API key then. I will name the file `openshift-e2e-ci-apikey`

Reply:
See https://github.com/openshift/release/pull/20555

Reply:
That sounds just fine, I'll update my PR to get that secret

Reply:
Updated to pull key from file, only if the provider is `ibmcloud` (and that secret is mounted as expected).

Reply:
https://github.com/openshift/release/pull/20433/files#diff-1a46129e809990d7d8a81a52a2a48bc8258fae24f171dccb14d3bf00301a3b35R121-R124